nimdoc/test_out_index_dot_html/expected/index.html

```diff
@@ -98,7 +96,7 @@ <h1 class="title">foo</h1>
   <ul class="simple simple-toc-section">
       <ul class="simple nested-toc-section">foo
       <li><a class="reference" href="#foo"
-    title="foo()">foo()</a></li>
+    title="foo()">foo</a></li>
```

Comment:
you're probably using an old nim to run `nim r nimdoc/tester.nim`; make sure you're using a nim build from your PR

Reply:
I was, thank you!

Reply:
I ran nim -r nimdoc/tester.nim with the latest devel(switched to using choosenim). Still these `()` remain. Huh